### ISSUE TITLE
Too few arguments to function ban_canLogin()

### ISSUE THREAD

**Issue by Hydrog3n** [kalvn/Shaarli-Material#65]
Hi, thank you for your work. 

I install your theme but on login page I have an error only with your theme. Have you some hint ?  

```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function ban_canLogin(), 0 passed in /shaarli/tmp/loginform.aeadca0864f49f08401fb0f97c162898.rtpl.php on line 5 and exactly 1 expected in /shaarli/index.php:383 
Stack trace: #0 /shaarli/tmp/loginform.aeadca0864f49f08401fb0f97c162898.rtpl.php(5): ban_canLogin()
#1 /shaarli/inc/rain.tpl.class.php(175): include('/shaarli/tmp/lo...') 
#2 /shaarli/application/PageBuilder.php(134): RainTPL->draw('loginform')
#3 /shaarli/index.php(800): PageBuilder->renderPage('loginform') 
#4 /shaarli/index.php(2219): renderPage(Object(ConfigManager), Object(PluginManager)) 
#5 {main} thrown in /shaarli/index.php on line 383
```

**Comment by kalvn** [comment]
Hello,
According to the error message, I suppose you are using Shaarli v0.8.3 and Shaarli Material v0.8.2 right?

Unfortunately, the way this version 0.8.3 was built doesn't allow me to release a proper version 0.8.3 of Shaarli Material.

Therefore, you have 2 options:

- Using the last master version of both [Shaarli](https://github.com/shaarli/Shaarli/archive/master.zip) and [Shaarli Material](https://github.com/kalvn/Shaarli-Material/archive/master.zip).
- Using [Shaarli v0.8.2](https://github.com/shaarli/Shaarli/releases/tag/v0.8.2) and [Shaarli Material v0.8.2](https://github.com/kalvn/Shaarli-Material/releases/tag/v0.8.2). If you go this way, just be aware of a problem that might appear with RSS and ATOM feeds with this version.

**Comment by ArthurHoaro** [comment]
Hey!

Shaarli v0.8.3 is just fixing a blocking issue with PHP 7.1. Using v0.8.2 won't fix it. Using both master branches might work, if you're up to date, but it isn't a really stable solution.

Maybe you could do as I did: commit a fix in your master branch, create a branch based on your v0.8.2 release, `git cherry-pick` the fix in this branch, and release a v0.8.3 from the new branch.

**Comment by Hydrog3n** [comment]
Exactly ! I didn't think about it at all ! I ready compatible with 0.8.3 and i install the release 0.8.2. 

Thank you for your help.

**Comment by kalvn** [comment]
@ArthurHoaro thanks for the hint, I didn't think of using a dedicated branch based on the v0.8.2. Since the fix was already done in the master branch, along with other fixes (I should definitely split my commits better), I couldn't reuse it. But I just created another one just for the branch.

@Hydrog3n thanks for the feedback, I'm glad it worked. Please just be aware of the ArthurHoaro's remark which is perfectly relevant. Thanks to him, I was now able to release a proper v0.8.3 so you should probably use it.

**Comment by Hydrog3n** [comment]
@ArthurHoaro It's work fine with my docker image with php 7.0 for the moment. So wait and see.

**Comment by ArthurHoaro** [comment]
Awesome! Glad I could help.

@Hydrog3n Well, that's weird, passing too few arguments is a breaking change from PHP 7.1. http://php.net/manual/fr/migration71.incompatible.php But as long as it's working, fine!

**Comment by Hydrog3n** [comment]
@ArthurHoaro After check version i have php 7.1 on my docker container. And with the master of Materiel it's ok. Maybe this error is only with v0.8.2 of Materiel. 
For now it's work so I don't touch anything ^^.

**Comment by kalvn** [comment]
Yes I think it works because master versions of both Shaarli and Shaarli Material implement the fix that was just done in v0.8.3.
